libraries/AP_HAL_ChibiOS/hwdef/X-MAV-AP-H743v2/hwdef-bl.dat

```diff
@@ -0,0 +1,43 @@
+# hw definition file for processing by chibios_hwdef.py
```

Comment:
you might consider adding the SD card interface in the bootloader to allow loading firmware from the SD card